css-viewport/Overview.bs

```diff
@@ -356,6 +360,149 @@ only the value previously set to it.
 }
 </pre>
 
+<h2 id='zoom-property'>
+	The 'zoom' property
+</h2>
+
+An element becomes zoomed when the 'zoom' property has a positive computed value different than 1.
+'zoom' affects computed property values and inherited property values.
+The computed value of 'zoom' is applied as a scalar to the used value of <<length>> CSS properties for the element and its descendants
+(including inherited properties such as 'line-height' or 'font-size'),
+plus intrinsic sizing of all replaced elements,
+background images,
+and nested frames (except for fenced frames [[!FENCED-FRAME]]),
+resulting in a magnification or minification effect.
+
+Nested values of 'zoom' multiply, resulting in additional scaling of <<length>> values.
+
+The 'zoom' property has no effect on <<length>> property values with computed values that are 'auto' or <<percentage>>.
+
+Note: Unlike 'transform',
+scaling the 'zoom' property affects layout.
+
+Note: The computed value of 'font-size' is never <<percentage>>;
+thus 'zoom' always applies.
+
+Note: 'zoom' does not affect or prevent 'transform' scaling.
+
+<pre class="propdef">
+	Name: zoom
+	Value: <<number>>
+	Initial: 1
+	Applies to: all <<length>> property values of block-level elements
+	Inherited: no
+	Percentages: Refer to all <<length>> property values
+	Media: visual
+	Computed value: as specified
+	Animation type: not animatable
+</pre>
+
+The values of this property have the following meanings:
+
+<dl dfn-for="zoom" dfn-type=value>
+	<dt><dfn><<number>></dfn>
+	<dd>
+		Positive floating point number indicating a zoom factor.
+		Numbers smaller than 1.0 indicate a "zoom out" or minification effect,
+		while numbers greater than 1.0 indicate a "zoom in" or magnification effect.
+
+	<dt><dfn><<percentage>></dfn>
+	<dd>
+		Positive floating point number,
+		followed by a percentage character ("%") which indicates a zoom factor multiplied by 100.
+</dl>
+
+Negative or zero values for 'zoom' are illegal.
```

Comment:
Negative value drops the declaration in WebKit and Chromium, but 0 doesn't.

http://software.hixie.ch/utilities/js/live-dom-viewer/saved/12260

Maybe instead zero should be defined to act as if it was 1 (or have no effect).

Reply:
Good catch. Done.